src/main/java/fr/adrienbrault/idea/symfony2plugin/config/ServiceReference.java

```diff
@@ -0,0 +1,59 @@
+package fr.adrienbrault.idea.symfony2plugin.config;
+
+import com.intellij.psi.PsiElement;
+import com.intellij.psi.PsiElementResolveResult;
+import com.intellij.psi.PsiPolyVariantReferenceBase;
+import com.intellij.psi.ResolveResult;
+import com.intellij.util.IncorrectOperationException;
+import fr.adrienbrault.idea.symfony2plugin.stubs.ServiceIndexUtil;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+
+public class ServiceReference extends PsiPolyVariantReferenceBase<PsiElement> {
```

Comment:
Ups... I see that similar class already exist in `fr.adrienbrault.idea.symfony2plugin.dic` package :sweat_smile: